src/modules/ID/index.ts

```diff
@@ -0,0 +1,19 @@
+export function getCollectionId(id: string): string | null {
+  let values = id.split(':')
+
+  if (values.length != 6) {
```

Comment:
The itemId has only two values. Referrer to https://github.com/decentraland/adr/blob/main/docs/ADR-42-third-party-collections-registry.md#items